guide/clear/clear-words.md

```diff
@@ -22,24 +22,21 @@ Use words that are common and clear to as many people as possible.
 
 ## What To Do
 
-- Use at the most common words and phrases. These are the terms that people with lower language skills are most likely to know
+WCAG section 3.1 addresses several related aspects of content readability (see below). This includes:
+
+- Use the most common words and phrases. These are the terms that people with lower language skills are most likely to know
 - Do not invent new words or give words new meanings that must be learned for the content
-- If you must use new terms, jargon, abbreviations, mnemonics or acronyms then make sure they are accompanied by an explanation
+- If you must use new terms, jargon, abbreviations, mnemonics or acronyms then make sure they are accompanied by an explanation. Consider the following approaches:
   - *? Use a twisty for the following (plus for all above bullets) or do we skip this level of detail ?*
-  - Add an simple language term in brackets next to it
-  - Consider a pop up definition
+  - Add explanation in brackets next to it
+  - Provide a pop up definition
```

Comment:
couldn't understand why this one in particular was "consider" while the others were "add" or "provide". Moved "consider" above to apply to all of them, and switched this to a more actionable word.

Reply:
Because, reasons.... I guess :)